spec/avrolution/compatibility_break_spec.rb

```diff
@@ -0,0 +1,108 @@
+describe Avrolution::CompatibilityBreak do
+  let(:name) { 'com.example.test' }
+  let(:fingerprint) do
+    Avro::Schema.parse({ name: name, type: :record }.to_json).sha256_resolution_fingerprint.to_s(16)
+  end
+
+  context "validation" do
+    let(:with_compatibility) { 'NONE' }
+    let(:after_compatibility) { nil }
+
+    subject { described_class.new(name, fingerprint, with_compatibility, after_compatibility) }
+
+    context "when name is blank" do
+      let(:name) { '' }
+
+      it { is_expected.to be_invalid }
+    end
+
+    context "when fingerprint is blank" do
+      let(:fingerprint) { nil }
+
+      it { is_expected.to be_invalid }
+    end
+
+    context "when with_compatibility does not have a valid value" do
+      let(:with_compatibility) { 'FOO' }
+
+      it { is_expected.to be_invalid }
+    end
+
+    context "when after_compatibility does not have a valid value" do
+      let(:after_compatibility) { 'BAR' }
+
+      it { is_expected.to be_invalid }
+    end
+  end
+
+  describe "#validate!" do
+    let(:with_compatibility) { 'NONE' }
+    let(:after_compatibility) { nil }
+
+    subject(:compatibility_break) do
```

Comment:
Should this be a `let` since we don't use the implicit subject? Same comment applies to a few other places in this file.